pkg/horizon/deployment.go

```diff
@@ -85,10 +96,11 @@ func Deployment(instance *horizonv1.Horizon, configHash string, labels map[strin
 							SecurityContext: &corev1.SecurityContext{
 								RunAsUser: &runAsUser,
 							},
-							Env:           env.MergeEnvs([]corev1.EnvVar{}, envVars),
-							VolumeMounts:  getVolumeMounts(),
-							Resources:     instance.Spec.Resources,
-							LivenessProbe: livenessProbe,
+							Env:            env.MergeEnvs([]corev1.EnvVar{}, envVars),
+							VolumeMounts:   getVolumeMounts(),
+							Resources:      instance.Spec.Resources,
+							ReadinessProbe: ReadinessProbe,
```

Comment:
upper case vs lower case in the variable declaration I believe is the only issue here.